frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -335,10 +339,11 @@ public void run() {
                         continue;
                     }
 
-                    if ("Torch worker started.".equals(result)) {
+                    Matcher matcher = PATTERN.matcher(result);
```

Comment:
Can we move `matcher` inside the `if` where it's used?

Reply:
The `matcher` is used in the `else if` clause and I didn't want to do the matching twice. Hence declared it outside